src/main/riddl/Venues/Locations/locationMessages.riddl

```diff
@@ -0,0 +1,58 @@
+command EstablishLocation is {
+  locationId: LocationId briefly "A unique identifier for this Location. This must be provided by the calling client.",
+  name: String,
+  description: String,
+  directions: String,
+  latLong: LatLong,
+  url: URL?,
+  capacity: Integer?,
+  accessible: Boolean,
+  amenities: String*,
```

Comment:
several attributes defined here are defined again in the Info type. I think we should remove the redundancy.